### PR TITLE
CVAT.js: Save and delete for shapes/tracks/tags

### PR DESCRIPTION
**Pull request by bsekachev**

### CODE REVIEW

File: cvatjs/.eslintrc.js [hunk 1]
```diff
@@ -48,5 +48,6 @@
         "indent": ["warn", 4],
         "no-useless-constructor": 0,
         "func-names": [0],
+        "valid-typeof": [0],
```
Comment (nmanovic) [inline cvatjs/.eslintrc.js]: why do we need the rule?

File: cvatjs/src/server-proxy.js [hunk 2]
```diff
@@ -179,10 +179,10 @@
             }
 
             async function logout() {
-                const { backendAPI } = window.cvat.config;
+                const host = window.cvat.config.backendAPI.slice(0, -7);
```
Comment (nmanovic) [inline cvatjs/src/server-proxy.js]: Why?

File: cvatjs/.eslintrc.js (hunk 1 above)
Comment (bsekachev) [reply cvatjs/.eslintrc.js]: @nmanovic 
In other case typeof works only with literals

File: cvatjs/src/server-proxy.js (hunk 2 above)
Comment (bsekachev) [reply cvatjs/src/server-proxy.js]: @nmanovic 
backendAPI is ```host/api/v1```
authorization url is ```host/auth```

**Pull request merged**